setup.py

```diff
@@ -29,5 +29,6 @@
         'postgres': ['psycopg2', 'SQLAlchemy'],
         'mysql': ['mysql-python', 'SQLAlchemy'],
         'azure': ['azure'],
+        'xml': ['dicttoxml'],
```

Comment:
oh, forgot to mention that `bz2file` should also be moved to an `extras_require` group.